File: 2026/Company/Company.Web/Modules/Administration/Tenants/TenantsColumns.cs
namespace Company.Administration.Forms;

[ColumnsScript("Administration.Tenants")]
[BasedOnRow(typeof(TenantsRow), CheckNames = true)]
public class TenantsColumns
{
    [EditLink]
    public int TenantId { get; set; }
    [EditLink, SortOrder(1)]
    public string TenantName { get; set; }
}

File: 2026/Company/Company.Web/Modules/ServerTypes/Administration/TenantsColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { TenantsRow } from "./TenantsRow";

export interface TenantsColumns {
    TenantId: Column<TenantsRow>;
    TenantName: Column<TenantsRow>;
}

export class TenantsColumns extends ColumnsBase<TenantsRow> {
    static readonly columnsKey = 'Administration.Tenants';
    static readonly Fields = fieldsProxy<TenantsColumns>();
}